includes/classes/order.php

```diff
@@ -412,6 +415,7 @@ protected function cart(shoppingCart $basket, $customer_id = null, $sendto_addre
           $taxCountryId = $tax_address->fields['entry_country_id'];
           $taxZoneId = $tax_address->fields['entry_zone_id'];
         }
+        $this->notify('NOTIFY_ORDER_CART_TAX_ADDRESS', $tax_address);
```

Comment:
Is the goal here to merely allow the observer to "inquire" or "watch" what the $tax_address query-result contains?  Or to also actually allow changes to any of the calculated values?  

If it's to also allow changes, I'd add some more parameters, such as `$taxCountryId, $taxZoneId` ... just as additional params, not as a nested array.